test/errname.js

```diff
@@ -0,0 +1,28 @@
+import os from 'os';
+import test from 'ava';
+import errname from '../lib/errname';
+
+const isWin = os.platform() === 'win32';
+
+// simulates failure to capture process.binding('uv');
+function fallback(code) {
```

Comment:
@kevva Join the discussion in https://github.com/sindresorhus/eslint-plugin-xo/issues/3 ;)